bin/git-lfs-s3

```diff
@@ -9,10 +9,16 @@ GitLfsS3::Application.set :aws_access_key_id, ENV['AWS_ACCESS_KEY_ID']
 GitLfsS3::Application.set :aws_secret_access_key, ENV['AWS_SECRET_ACCESS_KEY']
 GitLfsS3::Application.set :s3_bucket, ENV['S3_BUCKET']
 GitLfsS3::Application.set :server_url, ENV['LFS_SERVER_URL']
+GitLfsS3::Application.set :public_server, (ENV['LFS_PUBLIC_SERVER'] == 'true')
+GitLfsS3::Application.set :ceph_s3, (ENV['LFS_CEPH_S3'] == 'true')
 GitLfsS3::Application.set :logger, Logger.new(STDOUT)
 
-GitLfsS3::Application.on_authenticate do |username, password|
-  username == ENV['USERNAME'] && password == ENV['PASSWORD']
+GitLfsS3::Application.on_authenticate do |username, password, is_safe|
+  if is_safe # Whether the HTTP method is safe (GET, HEAD)
```

Comment:
See 9.1.1 in RFC-2616:
http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html

Also it's used in Django to indicate GET and HEAD methods (sometimes OPTION too). Example:

https://docs.djangoproject.com/en/1.8/topics/http/decorators/#django.views.decorators.http.require_safe